loafer/ext/aws/consumer.py

```diff
@@ -20,15 +20,16 @@ def __init__(self, source, endpoint_url=None, use_ssl=True, options=None, loop=N
         self._client = None
```

Comment:
The ```loop=None``` parameter and ```loop or asyncio.get_event_loop()``` are not required anymore, since we are replacing boto3 with aiobotocore.
It was only needed because the eventloop give us access to ```loop.run_in_executor```.